shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -152,12 +155,18 @@
     public static final String SLTS_SYSTEM_READ_BYTES = PREFIX + "segmentstore.storage.slts.system_read_bytes";     // Counter
     public static final String SLTS_SYSTEM_WRITE_BYTES = PREFIX + "segmentstore.storage.slts.system_write_bytes";   // Counter
     public static final String SLTS_CONCAT_BYTES = PREFIX + "segmentstore.storage.slts.concat_bytes";      // Counter
+
+    public static final String SLTS_READ_COUNT = PREFIX + "segmentstore.storage.slts.read_count";      // Counter
```

Comment:
What is the difference between these ant he ones 40 lines above? These ones have `.slts` in their name. 

If these are for the system segments/journal, let's name them as such. If they are for the metadata tables, let's name them as such. 

IMO, since we already have a lot of SLTS/Storage metrics, it is very important to clearly differentiate what they do by their name, otherwise someone unfamiliar with the inner workings of this will easily get confused.

Reply:
`segmentstore.storage.`  metrics record only for the individual single chunk storage call

`segmentstore.storage.slts.`  metrics record for entire end to end call to SLTS.
This is required because single SLTS call will often have multiple steps - access table segment metadata and potentially results in multiple calls to ChunkStorage. 
Rolling storage did not have these end to end metrics but we already have these second set for SLTS.

For some reason we did not have any metrics for count of read and write calls

Reply:
OK, I understand now. Would it make sense to rename the first one to include `chunk` in it?

Reply:
Agree. But may be later. 
I want to keep naming convention matching other metrics (Eg. concat/truncate/delete counts).
We should change name for all of those and also update existing dashboards etc. That would be a bigger and separate change in itself.